roles/test_operator/tasks/tempest-tests.yml

```diff
@@ -159,7 +159,8 @@
               item |
               combine({'tempestconfRun': {'overrides':
                 item.tempestconfRun.overrides + ' ' +
-                'whitebox_neutron_plugin_options.proxy_host_address ' + controller_ip
+                'whitebox_neutron_plugin_options.proxy_host_address ' + controller_ip + ' ' +
```

Comment:
it's going to be more and more complex. Maybe add a comment in L153 for others with example how it should look like. WDYT?

Reply:
Thank you @danpawlik for the review. this pr is no longer needed.